apps/src/templates/teacherDashboard/RecentCourses.jsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import SetUpMessage from './SetUpMessage';
+import FontAwesome from '../FontAwesome';
+import i18n from "@cdo/locale";
+import color from "../../util/color";
+
+const styles = {
+  section: {
+    width: 1600,
+  },
+  heading: {
+    paddingLeft: 20,
+    paddingRight: 10,
+    paddingTop: 10,
+    paddingBottom: 25,
+    marginLeft: 15,
+    fontSize: 20,
+    fontFamily: '"Gotham 3r", sans-serif',
+    zIndex: 2,
+    color: color.charcoal,
+    width: 1075
+  },
+  arrowIcon: {
+    paddingRight: 8
+  },
+  linkToViewAll: {
+    color: color.teal,
+    fontSize: 12,
+    fontFamily: '"Gotham", sans-serif',
+    fontWeight: 'bold',
+    marginTop: -2,
+    display: 'inline'
+  },
+  chevron: {
+    display: 'inline',
+    color: color.teal,
+    fontSize: 10,
+    fontWeight: 'bold',
+    marginLeft: 15,
+  },
+  linkBox: {
+    display: 'inline',
+    float: 'right',
+    textDecoration: 'none'
+  },
+};
+
+const RecentCourses = React.createClass({
+  propTypes: {
+    courses: React.PropTypes.array.isRequired
+  },
+
+  getInitialState() {
+    return {open: true};
+  },
+
+  renderArrowIcon() {
+    let icon = this.state.open ? 'caret-up' : 'caret-down';
+    return (
+     <FontAwesome icon={icon} style={styles.arrowIcon} onClick={this.toggleContent}/>
+    );
+  },
+
+  toggleContent() {
+    this.setState({open: !this.state.open});
+  },
+
+  renderContent() {
+    const { courses } = this.props;
+
+    if (this.state.open) {
+      if (courses.length > 0) {
+        return (
+          <div style={styles.content}>
+             {courses.slice(0,2).map(course => (
```

Comment:
React is complaining because I'm not giving each of these a unique key, but there isn't anything unique about the cards I'm passing in.  Do I need to generate a unique id for each card, kind of like we did with the tooltips then use that as the key? Or use the index?